scripts/overlap/kubectl-kubestellar-ensure-wmw

```diff
@@ -70,16 +70,16 @@ if [ "$KUBECONFIG_DIR" == "" ]; then
     mkdir -p $KUBECONFIG_DIR
     export KUBECONFIG_DIR
 fi
-if [ "$SPACE_MANAGER_KUBECONFIG" == "" ]; then
-    SPACE_MANAGER_KUBECONFIG="${KUBECONFIG_DIR}/config"
-    export SPACE_MANAGER_KUBECONFIG
+if [ "$SM_CONFIG" == "" ]; then
```

Comment:
(1) `commands.md` should document all the inputs.

(2) Inputs via environment variable are more troublesome than inputs on the command line.

(3) Can we simplify so that no special input is needed, caller should establish the fact that `kubectl` will access the SMAS?